### PR TITLE
Display timeout on progress bars instead of error

### PR DESCRIPTION
**Pull request by nbanmp**
Minor improvement, just getting familiar with the code.

### CODE REVIEW

**Comment by rocky** [comment]
@tagomaru Do you have any thoughts on this?

File: helpers.js [hunk 1]
```diff
@@ -292,9 +292,16 @@ const doAnalysis = async (client, config, jsonFiles, contractNames = null, limit
             if (progress) {
                 clearInterval(timer);
                 sleep.msleep(1000); // wait for last setInterval finising
-                bar.tick({
-                    'status': '✗ error'.red
-                });
+                // Check error message from armlet to determine if a timeout occured.
+                if(err.includes("User or default timeout reached after")) {
```
Comment (tagomaru) [inline helpers.js]: I guess the below error message should be hooked also.
https://github.com/ConsenSys/armlet/blob/master/lib/analysisPoller.js#L201-L202

Comment (tagomaru) [reply helpers.js]: thx!

**Comment by tagomaru** [comment]
@rocky 
LGTM now. Let's merge it!

**Comment by rocky** [comment]
Done. Thanks @nbanmp for this and thank @tagomaru for reviewing.

**Pull request merged**